src/subscription_manager/cert_sorter.py

```diff
@@ -289,9 +200,6 @@ def get_status_map() -> Dict[str, str]:
         # when function is called (not during start of application) due to
         # rhsm.service which can run for very long time
         status_map = {
-            VALID: _("Current"),
-            PARTIAL: _("Insufficient"),
-            INVALID: _("Invalid"),
             DISABLED: _("Disabled"),
```

Comment:
In the future we can rename this to "Registered" to be compatible with CLI, because it is only human readable string.